scripts/seed_users.js

```diff
@@ -312,7 +313,6 @@ function getClassroomsNamesFromUsers() {
 
 // Function to generate CSV of users
 function generateCSV() {
-    console.log("Generating CSV...");
     return new Promise(function(resolve, reject) {
         var csvWriter = createCsvWriter({
             path: filename + "_output.csv",
```

Comment:
I understand your idea to both add "output" and let CSV extension, but I'm not a big fan of a so long filename. I suggest to just add .out extension

Reply:
Are you suggesting to add `.out` extension to a `.csv` file? Can you give an example of input and output file names?

Reply:
Ideally the script should have something like a parameter "-o" to specify the output file. In the meantime you could just have an output file named "output.csv".

Reply:
For now, the output will be saved in "output.csv".
https://github.com/llaske/sugarizer-server/blob/50e1614975aaf332b7edcc0efc1f258089e9c228/scripts/seed_users.js#L329-L330

I was thinking about integrating Commander, but I guess it'll be overkill.